.github/CODEOWNERS

```diff
@@ -9,7 +9,8 @@ Jenkinsfile @brave/devops-team
 DEPS @brave/deps-reviewers
 script/deps.py @brave/deps-reviewers
 package*.json @brave/js-deps-reviewers
-**/Cargo.* @brave/rust-deps-reviewers
+**/Cargo.* @brave/rust-deps-reviewers @bridiver
+third_party/rust @brave/rust-deps-reviewers @bridiver
```

Comment:
you should just add yourself into `@brave/rust-deps-reviewers`. This syntax means that `@brave/rust-deps-reviewers` OR `@bridiver` is required for the PR to be merged.

Reply:
Added Brian to our Rust GitHub team.

Reply:
@goodov I specifically didn't do that because for now I want to make sure I see all of them and adding my self to @rust-deps-reviewers would allow other people to approve without me